sdk/batch/microsoft-azure-batch/src/test/java/com/microsoft/azure/batch/BatchIntegrationTestBase.java

```diff
@@ -87,6 +87,8 @@ private static void initTestMode() throws IOException {
                 testMode = TestBase.TestMode.RECORD;
             } else if (azureTestMode.equalsIgnoreCase("Playback")) {
                 testMode = TestBase.TestMode.PLAYBACK;
+            } else if (azureTestMode.equalsIgnoreCase("Live")) {
+                testMode = TestMode.RECORD;
             } else {
```

Comment:
Unrelated to this PR: If the env var is not a known value, we should fallback to PLAYBACK mode. This is the behavior in azure-core-test as well. So, we should have consistent behavior here.